zerver/openapi/markdown_extension.py

```diff
@@ -525,6 +528,8 @@ def render_title(self, function: str) -> List[str]:
         raw_title = get_openapi_summary(path, method)
         title.extend(raw_title.splitlines())
         title = ["# " + line for line in title]
+        if check_admin_config(path, method):
+            title.append("{!api-admin-only.md!}")
```

Comment:
This ends up on its own line because it's a list, right?

Reply:
Indeed -- the list contains each line that is supposed to be present( as can be seen in `title.extend(raw_title.splitlines())`)